main.ts

```diff
@@ -104,7 +106,41 @@ function fixBibleReferences(app) {
   }
 
   let wikiBible = function(b: { book: any; chapter: any; verse: any; ref: any; }) {
-    return `[[ESV/${b.book}/${b.book}-${b.chapter}${prefix(b.verse, "#")}|${b.ref}]]`;
+    /*
+      Replaces a custom pattern like "{{book}}/{{book}}-{{chapter}}#{{verse}}|{{input}}"
+      with the variables stored.
+    */
+     
+    // There is surely a prettier way to do this – maybe objects?
+    let toReplace = ["{{book}}", "{{chapter}}", "{{verse}}", "{{input}}"]
+    let replaceWith = [`${b.book}`, `${b.chapter}`, `${b.verse}`, `${b.ref}`]

```

Comment:
As I mentioned in the loom… you probably can just use

```typescript
let replaceWith = [b.book, b.chapter, b.verse, b.ref]
```

b.book is a string (like `"Genesis"`) so wrapping it in another template string where it echoes itself is unneeded.

Man this is all exciting! Great work!